src/utils/exportSQL/postgres.js

```diff
@@ -31,7 +31,9 @@ export function toPostgres(diagram) {
             (field) =>
               `${exportFieldComment(field.comment)}\t"${
                 field.name
-              }" ${field.type}${field.isArray ? " ARRAY" : ""}${field.notNull ? " NOT NULL" : ""}${field.unique ? " UNIQUE" : ""}${
+              }" ${field.type}${
+                field.size !== undefined && field.size !== "" ? "(" + field.size + ")" : ""
```

Comment:
```suggestion
                field.size ? "(" + field.size + ")" : ""
```
isn't this enough? or we need to test against undefined/empty for some reason?

Reply:
Hi, I just copy this from https://github.com/drawdb-io/drawdb/blob/main/src/utils/exportSQL/mysql.js#L12

Reply:
Not all fields have a size so yeah